conf/cassandra/migration/default/V2__query_alerts.cql

```diff
@@ -0,0 +1,56 @@
+-- description: creates initial schema
+-- authoredAt: 1370028268000
+-- up:
+
+-- stage: 1
+DROP MATERIALIZED VIEW alerts_by_cluster;
+
+-- stage: 2
+DROP MATERIALIZED VIEW alerts_by_organization;
+
+-- stage: 3
+ALTER TABLE alerts DROP (cluster, service, context, metric, statistic, operator, quantity_value, quantity_unit);
+
+-- stage: 4
+ALTER TABLE alerts ADD (query varchar);
+
+-- stage: 5
+CREATE MATERIALIZED VIEW alerts_by_organization AS
+SELECT *
+FROM alerts
+WHERE organization IS NOT NULL AND uuid IS NOT NULL
+PRIMARY KEY (organization, uuid);
+
+-- down:
+
+--stage: 1
+DROP MATERIALIZED VIEW alerts_by_organization;
+
+-- stage: 2
+ALTER TABLE alerts DROP (query);
+
+-- stage: 3
+ALTER TABLE alerts add (
+    cluster varchar,
```

Comment:
I didn't think this was still a property of the alert. (along with others)

Reply:
You're right.  This is the downs part.  You have to undo what you did in the ups.